G/Gnuplot/build_tarballs.jl

```diff
@@ -1,35 +1,38 @@
-
 using BinaryBuilder, Pkg
 
 name = "Gnuplot"
-version = v"5.4.5"
+version = v"5.4.6"
```

Comment:
If you're faking the version number anyway, why not sticking to 5.4.5?

Reply:
I'm fine with either 5.4.5 or bump to the latest 5.4.10.